react-native-git-upgrade/cliEntry.js

```diff
@@ -0,0 +1,239 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import shell from 'shelljs';
+import Promise from 'promise';
+import yeoman from 'yeoman-environment';
+import TerminalAdapter from 'yeoman-environment/lib/adapter'
+import log from 'npmlog';
+import rimraf from 'rimraf';
```

Comment:
Seems like we use `require` everywhere else in the CLI code. Could you use it please for consistency since we'll be adding this code to the CLI too? The reason to add this code to the CLI is people will just need to run 'react-native upgrade' without installing packages to get the awesome git-based upgrades.

    const chalk = require('chalk');